hack/gen-olm-artifacts.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
```

Comment:
Just a thought, not an issue here, but should we think about moving towards Go as a single language for stuff like this? It can be run like a scripting language `go run gen-olm-artifacts.go` as well as compiled, and the templates package makes this kind of thing pretty easy.

Reply:
my only reservation would be if we want to allow someone to utilize without relying on golang to accomplish deployment :man_shrugging: